Resources/PO/TopNav.robot

```diff
@@ -0,0 +1,45 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+
+*** Variables ***
+${LOGO_LOCATOR} =  //li[@class="logo"]
+${SIGNIN_LINK_LOCATOR} =  //li[@class="user-li"]/a
+${USER_MENU_LOCATOR} =  //div[@id="user-dropdown"]
+${USER_PROFILE_LINK} =  //a[@href="/home"]
+#user-dropdown-menu > li:nth-child(4) > a
+
+*** Keywords ***
+Verify Logo Loaded
+    Wait Until Page Contains Element  ${LOGO_LOCATOR}
+
+Navigate To Main Page
+    Verify Logo Loaded
+    Click Element  ${LOGO_LOCATOR}
+
+Go to Login Page
+    Wait Until Page Contains Element  ${SIGNIN_LINK_LOCATOR}
+    Click Element  ${SIGNIN_LINK_LOCATOR}
+
+Verify User Is Signed In
+    TopNav.Verify User Menu Exists
+
+Verify User Menu Exists
```

Comment:
now `Verify User Menu Exists` and `Verify User Is Signed In` looks like almost the same keywords. What's  the reason to split them ?